### PR TITLE
add cantamen provider stadtmobil_rhein-neckar

### PR DESCRIPTION
**Pull request by ThorstenFroehlinghaus**
stadtmobil_rhein-neckar is fully configured (incl. pricing), while the other cantamen providers contain no pricing yet.
I tested stadtmobil_rhein-neckar on test-ipl using "python3 -m x2gbfs.x2gbfs -p stadtmobil_rhein-neckar -b 'file:out' ".
After being merged, I will configure stadtmobil_rhein-neckar in lamassu.

### CODE REVIEW

**Review by hbruch — COMMENTED** [review]
I'd prefer to do limit this PR to stadtmobil_rhein-neckar only, and add the others completely configured with corresponding CHANGELOG and PRs. Now the PR reads "stadtmobil_rhein-neckar" but contains preparations for other providers as well.

**Comment by ThorstenFroehlinghaus** [comment]
Ok, I removed the config files of the other providers.

File: x2gbfs/x2gbfs.py [hunk 1]
```diff
@@ -45,7 +45,7 @@ def build_extractor(provider: str, feed_config: Dict[str, Any]) -> BaseProvider:
 
         return VoiRaumobil(api_url, api_user, api_password)
     if (
-        provider in ['my-e-car', 'oekostadt_renningen']
+        provider in ['my-e-car', 'oekostadt_renningen', 'swu2go', 'conficars_ulm', 'gruene_flotte_freiburg']
```
Comment (hbruch) [inline x2gbfs/x2gbfs.py:48]: Do you want to shift this addition to another PR as well?

**Review by hbruch — APPROVED** [review]
LGTM

**Pull request merged**